### PR TITLE
Auth methods changelog updates

### PR DESCRIPTION
**Pull request by mmcla**
Changelog updates for the changes in https://github.com/microsoftgraph/microsoft-graph-docs/pull/11058/

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+          "ChangedApiName": "authenticationMethod,passwordAuthenticationMethod,phoneAuthenticationMethod,emailAuthenticationMethod,passwordlessMicrosoftAuthenticatorAuthenticationMethod",
+          "ChangeType": "Change",
+          "Description": "Updated permissions to reflect the ability of users to call the APIs on themselves and for admins to use app-only permissions on all [authentication methods](https://docs.microsoft.com/en-us/graph/api/resources/authenticationmethod?view=graph-rest-beta).",
+          "Target": "authenticationMethod,passwordAuthenticationMethod,phoneAuthenticationMethod,emailAuthenticationMethod,passwordlessMicrosoftAuthenticatorAuthenticationMethod"
+        }
+      ],
+      "Id": "151b955d-fe34-496d-a76c-7cf7865da63d",
+      "Cloud": "prd",
+      "Version": "beta",
+      "CreatedDateTime": "2020-12-16T00:00:00.000Z",
```
Comment (Lauragra) [inline changelog/Microsoft.DirectoryServices.json:26]: The date here should be the date that we publish - i.e. today.

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+          "Target": "authenticationMethod,passwordAuthenticationMethod,phoneAuthenticationMethod"
+        },
+        {
+          "Id": "b3b25845-1cec-49be-b49f-5b51123e43aa",
+          "ApiChange": "Resource",
+          "ChangedApiName": "authenticationMethod,passwordAuthenticationMethod,phoneAuthenticationMethod,emailAuthenticationMethod,passwordlessMicrosoftAuthenticatorAuthenticationMethod",
+          "ChangeType": "Change",
+          "Description": "Updated permissions to reflect the ability of users to call the APIs on themselves and for admins to use app-only permissions on all [authentication methods](https://docs.microsoft.com/en-us/graph/api/resources/authenticationmethod?view=graph-rest-beta).",
+          "Target": "authenticationMethod,passwordAuthenticationMethod,phoneAuthenticationMethod,emailAuthenticationMethod,passwordlessMicrosoftAuthenticatorAuthenticationMethod"
```
Comment (Lauragra) [inline changelog/Microsoft.DirectoryServices.json:20]: Technically, Target should be a single value according to the guidance. This hasn't been strictly enforced, but to support a future feature to enable filtering at the API level, the recommendation is one separate entry per API change. See the guidance here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog?anchor=changelist-record-properties

File: changelog/Microsoft.DirectoryServices.json [hunk 3]
```diff
@@ -1,5 +1,32 @@
 {
   "changelog": [
+    {
+
+      "ChangeList": [
+        {
+          "Id": "78f63c95-8c9e-4438-ad45-63f6174a4e33",
+          "ApiChange": "Resource",
+          "ChangedApiName": "windowsHelloForBusinessAuthenticationMethod, microsoftAuthenticatorAuthenticationMethod",
```
Comment (Lauragra) [inline changelog/Microsoft.DirectoryServices.json:9]: ChangedApiName should also be a single value.

File: changelog/Microsoft.DirectoryServices.json (hunk 1 above)
Comment (mmcla) [reply changelog/Microsoft.DirectoryServices.json]: Fixed

File: changelog/Microsoft.DirectoryServices.json (hunk 2 above)
Comment (mmcla) [reply changelog/Microsoft.DirectoryServices.json]: Fixed

File: changelog/Microsoft.DirectoryServices.json (hunk 3 above)
Comment (mmcla) [reply changelog/Microsoft.DirectoryServices.json]: Fixed

**Pull request merged**